vue3_ts/src/core/utils/index.ts

```diff
@@ -1 +1,2 @@
 export * from './example.utils'
+export * from './yup'
```

Comment:
utils folder created for your custom code utils. 3rd party packages should be placed to: `src/plugins/yup/index.ts`